### DISCUSSION TITLE
New-lines in text description disappear

### DISCUSSION THREAD

**Discussion by bsels** [dodona-edu/feedback-and-support#2] [Questions - Vragen] [answered]
New-lines in testcase text description disappears:
`{"description": {"description": "100\n53\n<", "format": "text"}, "command": "start-testcase"}`
![Schermafdruk van 2020-12-04 09-54-26](https://user-images.githubusercontent.com/38499296/101144111-2141af80-3618-11eb-9f33-d0605192b371.png)

**Comment by pdawyndt** [comment]
@chvp Is this due to the fact that the Python judge has its own renderer or is it related to how the description of the test (in this case what appears on `stdin` is formatted? After all, the representation of the feedback for this [same exercise](https://dodona.ugent.be/nl/activities/1047652305/) with the Python judge does show the `stdin` multline.

![image](https://user-images.githubusercontent.com/5736113/116784123-c4490900-aa92-11eb-8356-d8435466f060.png)

**Comment by chvp** [comment]
This is due to the difference in the `format` property. `"text"` will be inserted in the feedback table as plain text, which means that the browser will do whitespace normalization.

**Comment by pdawyndt** [comment]
So better to use `html` as a format, and then include the description as a `<pre>`-formatted message? In the feedback table of the Python judge, it is formatted as
```html
<span class="code">100
53
&lt;
</span>
```

**Comment by chvp** [comment, marked as answer]
No. `html` formatting is expensive for the renderer. A `code` format exists for this usecase.
